sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -404,8 +404,7 @@ void guidance_h_nav_enter(void)
   /* horizontal position setpoint from navigation/flightplan */
   guidance_h_set_pos(nav.carrot.y, nav.carrot.x);
   reset_guidance_reference_from_current_position();
-  /* set nav_heading to current heading */
-  nav.heading = stateGetNedToBodyEulers_f()->psi;
+
   guidance_h_set_heading(nav.heading);
   /* call specific implementation */
   guidance_h_run_enter();
```

Comment:
if the `nav.heading` is now set from the `guidance_h_run_enter()` function, maybe the last two lines should be swapped.

Reply:
Good point, should be fixed now!